ts/components/leftpane/overlay/choose-action/OverlayChooseAction.tsx

```diff
@@ -0,0 +1,96 @@
+import React from 'react';
+// tslint:disable: use-simple-attributes no-submodule-imports
+
+import { useDispatch } from 'react-redux';
+import { resetOverlayMode, setOverlayMode } from '../../../../state/ducks/section';
+import useKey from 'react-use/lib/useKey';
+import styled from 'styled-components';
+import { SessionIcon, SessionIconType } from '../../../icon';
+import { ContactsListWithBreaks } from './ContactsListWithBreaks';
+
+const StyledActionRow = styled.button`
+  border: none;
+  display: flex;
+  align-items: center;
+  border-bottom: 1px var(--color-session-border) solid;
```

Comment:
Kind of surprised CSS is intelligent enough to not fail with the color and the border style being swapped. You don't need to change anything.

i.e. normal approach is `border-bottom: 1px solid var(--color-session-border);`